R/finalize_explanation.R

```diff
@@ -153,3 +168,238 @@ compute_shapley_new <- function(internal, dt_vS) {
 
   return(dt_kshap)
 }
+
+
+
+#' Mean Squared Error of the Contribution Function `v(S)`
+#'
+#' @param internal List with the different parameters, data and functions used internally in the [explain()] function.
```

Comment:
Do not rewrite documentation of common arguments, use #' @inheritParams default_doc instead.

Reply:
I had forgotten about that one. Thanks!